### DISCUSSION TITLE
Problems with using arrow keys on Pulumi over ssh

### DISCUSSION THREAD

**Discussion by agluck91** [blinksh/blink#1856] [Support]
### Checklist

- [ x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [ x] Read the [docs](http://blink.sh/docs/)
- [ x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

Ipad OS 16 and 17 both
Current version.
SSH to M1 MacBook Pro on same network

### Describe the bug

When using interactive programs like pulumi, the program asks you to navigate a menu with the arrow keys. When I use the arrows keys it crashes pulumi with this error. Pulumi is just an example, other programs do this too. Something is not being sent to the command line the same way it is on other shells. I do not have this problem on item or warp. There is no error that happens, the program just acts like I pressed enter without selecting anything. 

<img width="1062" alt="image" src="https://github.com/blinksh/blink/assets/113385018/d0be253c-d558-4491-b91a-a630edf612d0">

Program will navigate up to the next selection without crashing.

**Comment by yury** [comment]
Confirmed, investigating further...

As a workaround you can run `tmux` and run  `pulumi` commands from tmux session...

**Comment by yury** [comment]
Ok, found related issue in `pulumi` project pulumi/pulumi/issues/13733
Also this one is related too go-survey/survey/issues/341

I think, fastest way to fix this issue it to upvote pulumi/pulumi/issues/13733

**Comment by agluck91** [comment]
Daaang, Niiice Yury. I have been considering going full on tmux anyway

I will also upvote the issue with pulumi! Thanks again!

**Comment by carloscabanero** [comment]
Changing the title to specify it is Pulumi only, and moving to Discussion as this is informative.

From our side, this is related to #1836.
